library/src/main/java/com/joom/grip/io/JrtFileSource.kt

```diff
@@ -0,0 +1,21 @@
+package com.joom.grip.io
+
+import java.nio.file.Files
+import java.nio.file.Path
+import kotlin.io.path.extension
+
+class JrtFileSource(private val jrtPath: Path) : FileSource {
+
+  override fun listFiles(callback: (name: String, type: FileSource.EntryType) -> Unit) {
+    Files.walk(jrtPath)
+      .filter { it.extension.endsWith("class") }
```

Comment:
Why is it `extension.endsWith("class")`? Shouldn't it be `it.extension == "class"`?

Reply:
yep, good notice